keadm/cmd/keadm/app/cmd/common/types.go

```diff
@@ -30,15 +30,16 @@ type InitOptions struct {
 	TarballPath      string
 }
 
-//BetaInitOptions has the kubeedge cloud init-beta information filled by CLI
-type BetaInitOptions struct {
+//InitBetaOptions has the kubeedge cloud init-beta information filled by CLI
+type InitBetaOptions struct {
 	KubeConfig       string
+	KubeEdgeVersion  string
 	AdvertiseAddress string
 	Manifests        string
 	Namespace        string
-	CloudcoreImage   string
+	CloudcoreRepo    string
 	CloudcoreTag     string
-	IptablesMgrImage string
+	IptablesMgrRepo  string
```

Comment:
These two options haven't exposed in flag?

Reply:
Yes. We can just use --set falg to confirm it.

Reply:
Using --set flag to set value in chart directly?

Reply:
> Using --set flag to set value in chart directly?

Yes. The user can input their own tag or repo.

Reply:
Ok, we need more docs. And the options here can  be removed?

Reply:
Yes, i will fix it

Reply:
Add ref: https://github.com/kubeedge/kubeedge/tree/master/charts/cloudcore/README.md